pkg/network/vmispec/network_test.go

```diff
@@ -55,6 +55,39 @@ var _ = Describe("Network", func() {
 				multusSecondaryNetwork3,
 			}),
 	)
+
+	DescribeTable("should fail to return the default network", func(inputNetworks []v1.Network, expectNetwork *v1.Network) {
+		defaultNetwork, err := vmispec.LookUpDefaultNetwork(inputNetworks)
+
+		Expect(err).To(HaveOccurred())
+		Expect(defaultNetwork).To(Equal(expectNetwork))
+	},
+		Entry("when there are no networks", []v1.Network{}, nil),
```

Comment:
Does it make sense to add a scenario where `[]v1.Network` is `nil` (similar to CreateNetworkNameScheme test suite)?

Reply:
I think it is redundant to have two, this is a language feature (to behave similar to `nil` and an empty list.
But if the author wants to be pedantic, a `nil` may represent a more realistic usage.
IMO it is a nit.